crates/astria-core/src/sequencer/v1alpha1/transaction/mod.rs

```diff
@@ -4,6 +4,7 @@ use ed25519_consensus::{
     VerificationKey,
 };
 use prost::Message as _;
+use sha2::Sha256;
```

Comment:
nit: Would move this into the `use` statements of the method because it's only used there.